.github/workflows/pr-verify.yml

```diff
@@ -23,11 +23,11 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-maven-
     - name: Verify formatting
-      run: mvn -B formatter:validate
+      run: mvn -B formatter:validate impsort:check
     - name: Build
-      run: mvn -B clean install -DskipTests  
+      run: mvn -B clean install -Pquick,\!formatting-check
     - name: Run tests
-      run: mvn -B verify -Pcompliance --file pom.xml
+      run: mvn -B verify -Pcompliance,\!formatting-check --file pom.xml
```

Comment:
I am not 100% the backslash is necessary. You need it if you run this on the command line but I don't know how clever the github actions are with this kind of thing.